tox.ini

```diff
@@ -19,10 +19,10 @@ max_issue_threshold=1
 [testenv]
 usedevelop=True
 commands=
-    core: pytest {posargs:tests/core}
-    interop: pytest {posargs:tests/interop}
+    core: pytest -n auto {posargs:tests/core}
+    interop: pytest -n auto {posargs:tests/interop}
     docs: make check-docs-ci
-    demos: pytest {posargs:tests/core/examples/test_examples.py}
+    demos: pytest -n auto {posargs:tests/core/examples/test_examples.py}
```

Comment:
Reduces the CI time by half:
<img width="881" alt="image" src="https://github.com/user-attachments/assets/a18439fa-e0cd-4512-a9ff-077536cac482" />